tests/i2c_end_to_end_sim/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(i2c_end_to_end_sim)
+
+# root
+set(XMOS_SANDBOX_DIR ${CMAKE_SOURCE_DIR}/../../..)
+
+# target: choose the target platform
+set(APP_HW_TARGET XCORE-AI-EXPLORER)
+
+# Includes and flags
+set(APP_INCLUDES src)
+set(APP_COMPILER_FLAGS
+  -O2
+  -g
+  -report
+  -Wall
+  -DUSE_I2C=1
+)
+set(APP_C_SRCS ../../lib_device_control/host/device_access_i2c_xcore.xc ../../lib_device_control/host/util.c)
```

Comment:
is not lib_device_control/lib_build_info.cmake already including these files? (maybe Im wrong)

Reply:
no, lib_build_info.cmake only includes the files used in the device-side library. Those files are part of the host side and they are not included, as we don't use xmake for the host applications:  the XC I2C host code is the only exception.